package/MDAnalysis/analysis/dihedrals.py

```diff
@@ -173,32 +173,59 @@
    :members:
    :inherited-members:
 
-   .. attribute:: angles
+   .. attribute:: results.angles
 
        Contains the time steps of the angles for each atomgroup in the list as
        an ``n_frames×len(atomgroups)`` :class:`numpy.ndarray` with content
        ``[[angle 1, angle 2, ...], [time step 2], ...]``.
 
+   .. attribute:: angles
+
+       Alias to the :attr:`results.angles` attribute.
+
+       .. deprecated:: 2.0.0
+          Will be removed in MDAnalysis 3.0.0. Please use
+          :attr:`results.angles` instead.
+
+
 .. autoclass:: Ramachandran
    :members:
    :inherited-members:
 
-   .. attribute:: angles
+   .. attribute:: results.angles
 
        Contains the time steps of the :math:`\phi` and :math:`\psi` angles for
        each residue as an ``n_frames×n_residues×2`` :class:`numpy.ndarray` with
        content ``[[[phi, psi], [residue 2], ...], [time step 2], ...]``.
 
```

Comment:
add versionadded:: 2.0.0

Reply:
There's a `versionchanged` that mentions it below (mainly bringing this up because I don't think I've been adding `versionadded` in the previous PRs), but we can start doing this if you want too (it's not exactly a lot of work).

Reply:
I know and that's good. I find that the versionadded completes the picture. I put it in the general instructions in #3261 . We can update whenever, though.